### PR TITLE
Update README.md with lots of data-splaining and contributing info

### PR DESCRIPTION
**Pull request by mdittmer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,3 +12,170 @@ the platform.
...
+The Failure to Ship metric counts APIs that The Browser does not provide, but
+all other browsers provide after the Grace Period.
+
+#### Rationale
+
+Failing to ship an API that other major vendors provide requires web developers
+to use special code paths to remain interoperable. Smaller values are good;
+larger values are bad.
+
+#### Definition
```
Comment (arobins) [inline README.md]: Missing section header

File: README.md [hunk 2]
```diff
@@ -12,3 +12,170 @@ the platform.
...
+The Aggressive Removal metric counts APIs that The Browser once provided, but no
+longer does, and all other browsers provide after the Grace Period.
+
+#### Rationale
+
+Removing an API from only one browser risks breaking existing sites that
+(reasonably) assume that all browsers support the API. Smaller values are good;
+larger values are bad.
+
+#### Definition
```
Comment (arobins) [inline README.md]: Missing section header

File: README.md [hunk 3]
```diff
@@ -12,3 +12,170 @@ the platform.
...
+
+### Collecting data
+
+**NOTE**: The current data collection process requires
+a [BrowserStack](https://www.browserstack.com/)
+account, [Cloud Datastore](https://cloud.google.com/datastore/) credentials, two
+separate `git clone`s, and a whole lot of RAM. We hope to streamline and
+simplify this process soon. If you have all the prerequisites, read on…
+
+1. Clone [mdittmer/web-apis](https://github.com/mdittmer/web-apis) and follow
```
Comment (arobins) [inline README.md]: Is there a plan to move this into the GoogleChrome organization at some point?

File: README.md [hunk 4]
```diff
@@ -12,3 +12,170 @@ the platform.
...
+   the
+   [data collection instructions](https://github.com/mdittmer/web-apis#setup-browserstack) for
+   *historical data collection using BrowserStack*.
+
+2. Copy `/path/to/mdittmer/web-apis/data/og/*.json` to
+   `/path/to/GoogleChrome/confluence/data/og`.
+
+3. In `/path/to/GoogleChrome/confluence`, run `node --max_old_space_size=16384
```
Comment (arobins) [inline README.md]: Missing step: how to configure the local instance to talk to cloud datastore

File: README.md (hunk 3 above)
Comment (mdittmer) [reply README.md]: Yeah, I guess. It depends on how high priority it is to make it *easy* (not just *possible*) for third parties to reproduce results.

File: README.md (hunk 4 above)
Comment (mdittmer) [reply README.md]: Done.

File: README.md (hunk 2 above)
Comment (mdittmer) [reply README.md]: Done.

File: README.md (hunk 1 above)
Comment (mdittmer) [reply README.md]: Done.

**Comment by mdittmer** [comment]
This will fix #98

File: README.md [hunk 5]
```diff
@@ -12,3 +12,179 @@ the platform.
...
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+- [The Catalog](#the-catalog)
+- [The Metrics](#the-metrics)
+  - [API Velocity](#api-velocity)
+    - [Definition](#definition)
+    - [Rationale](#rationale)
+  - [Failure to Ship](#failure-to-ship)
+    - [Definition](#definition-1)
```
Comment (arobins) [inline README.md]: Is there a way of giving these sections better tags? Links into this document will break if things are reordered.

**Pull request merged**